UPGRADING.INTERNALS

```diff
@@ -26,6 +26,10 @@ PHP 8.3 INTERNALS UPGRADE NOTES
   break the build of third-party extensions which relied on this
   implementation detail.  Those extensions may need to add the missing
   #include lines.
+* Since version 8, PHP requires a C99 compiler.  Configure-time checks
```

Comment:
FWIW, we do not actually fully C99 compliant compilers, but merely rely on some of its features (e.g. we do not require VLA support, which is optional as of C11). I don't think this needs to be updated, though.

Reply:
Requiring a C99 compiler doesn't imply that all C99 features are really used - VLA is a special example of a feature that many people nowadays think was a bad idea, and I'm happy PHP doesn't use them.
I compile my projects with `-Wvla` so GCC kicks me if I accidently use one...

Reply:
> we do not require VLA support, which is optional as of C11

@cmb69, turns out PHP does use VLAs!
https://github.com/php/php-src/blob/2f65da0080d5213d6a07ff40ae8a1446a457605d/sapi/fpm/fpm/fpm_status.c#L64
https://github.com/php/php-src/blob/2f65da0080d5213d6a07ff40ae8a1446a457605d/sapi/fpm/fpm/fpm_stdio.c#L173
https://github.com/php/php-src/blob/2f65da0080d5213d6a07ff40ae8a1446a457605d/sapi/fpm/fpm/fpm_php_trace.c#L43